client/src/index.css

```diff
@@ -148,7 +148,7 @@ fieldset.danger {
 }
 
 #duration {
-	width: 10em;
+	width: 100%;
```

Comment:
The duration field is way too wide:
![Screenshot from 2020-11-26 14-22-18](https://user-images.githubusercontent.com/445238/100355994-ed9ccf00-2ff2-11eb-9634-4f2c905cf73a.png)